.github/pull_request_template.md

```diff
@@ -0,0 +1,4 @@
+### Checklist
+- [ ] Relevant issue is linked
+- [ ] Docs updated/issue for docs created
+- [ ] Added relevant tests
```

Comment:
Alongside the checklist I'd include a "template" for linking issue so it's even harder to forget and so we follow the same format in every PR. Like

```suggestion
Closes 

### Checklist
- [ ] Relevant issue is linked
- [ ] Docs updated/issue for docs created
- [ ] Added relevant tests
```

Reply:
I don't think that's necessary, one might link with the UI directly

Reply:
Didn't think of the UI. If we prefer it for linking the issues, current template is fine.

Reply:
I prefer UI, there is a reminder about linking the issue already